dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-fixed-fee.ts

```diff
@@ -0,0 +1,36 @@
+import { differenceInDays } from 'date-fns';
+import { CalculateFixedFeeUtilisationReportParams } from '@ukef/dtfs2-common';
+import { calculateFixedFeeFromDaysRemaining } from '../../../../../helpers/calculate-fixed-fee-from-days-remaining';
+
+/**
+ * getNumberOfDaysInCoverPeriod
+ * calculates number of days in the cover period between the cover start and end dates
+ * @param coverStartDate
+ * @param coverEndDate
+ * @returns number of days in the cover period
+ */
+export const getNumberOfDaysInCoverPeriod = (coverStartDate: Date, coverEndDate: Date): number => differenceInDays(coverEndDate, coverStartDate);
+
+/**
+ * calculateFixedFee
+ * Calculates the fixed fee for the utilisation report
+ * gets the number of days in the cover period
+ * calculates and returns the fixed free from calculateFixedFeeFromDaysRemaining
+ * @param utilisation - The facility utilisation
+ * @param coverStartDate - The facility cover start date
+ * @param coverEndDate - The facility cover end date
+ * @param interestPercentage - The facility interest percentage
+ * @param dayCountBasis - The facility day count basis
+ * @returns The fixed fee for the current report period
+ */
+export const calculateFixedFee = ({
```

Comment:
We already have this function in dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/calculate-fixed-fee.ts. 
If this is going to be the function that calculates initialFixedFee for a new facility can we call it something like that e.g. calculateInitialFacilityFixedFee?
We should probably update the documentation at the top as well to be 
`Calculates the fixed fee for a facility` or something as it's facility based not utilisation report based

Reply:
changed the name